doc/deploy.md

```diff
@@ -0,0 +1,41 @@
+Deployment to Elastic Beanstalk
+===============================
+
+Tools required:
+
+* git
+* gpg
+* node and npm
+* zip
+
+This guide is a bit terse and assumes that your hard drive is encrypted.  We don't want encryption keys leaking out to the world because a laptop was stolen.
+
+1. Clone the repository somewhere new.
+
+        git clone https://github.com/opentoken-io/opentoken.git release
+
+2. Install modules.
+
+        cd release
+        npm install
+
+3. Decrypt the encryption key.
+
+        gpg --decrypt encryption.key.asc > encryption.key
+
+4. Make sure tests pass.
+
+        npm test
+
+5. If that fails, **stop here**.
```

Comment:
I somewhat hope that the person doing the deploy does not need to have that spelled out for them.  Am I assuming too much?